packages/components/pie-input/test/component/pie-input.spec.ts

```diff
@@ -1206,4 +1223,124 @@ test.describe('PieInput - Component tests', () => {
             });
         });
     });
+
+    test.describe('Attributes:', () => {
+        test.describe('aria-describedby', () => {
+            test.describe('when `assistiveText` is not defined', () => {
+                test('should not render the attribute', async ({ mount }) => {
+                    // Arrange
+                    const component = await mount(PieInput, {
+                        props: {} as InputProps,
+                    });
+
+                    // Act
+                    const input = component.locator('input');
+
+                    const componentAttribute = await input.getAttribute('aria-describedby');
+
+                    // Assert
+                    expect(componentAttribute).toBeNull();
+                });
+            });
+
+            test.describe('when `assistiveText` is defined', () => {
+                test('should render the attribute correctly with the correct value', async ({ mount }) => {
+                    // Arrange
+                    const component = await mount(PieInput, {
+                        props: {
+                            assistiveText: 'Some useful message',
+                        } as InputProps,
+                    });
+
+                    // Act
+                    const input = component.locator('input');
+
+                    const componentAttribute = await input.getAttribute('aria-describedby');
+
+                    // Assert
+                    expect(componentAttribute).toBe('assistive-text');
+                });
+            });
+        });
+
+        test.describe('aria-invalid', () => {
+            test.describe('when the component status is set to `error`', () => {
+                test('should render the `aria-invalid` attribute', async ({ mount }) => {
+                    // Arrange
+                    const component = await mount(PieInput, {
+                        props: {
+                            status: 'error',
+                        } as InputProps,
+                    });
+
+                    // Act
+                    const input = component.locator('input');
+
+                    const componentAttribute = await input.getAttribute('aria-invalid');
+
+                    // Assert
+                    expect(componentAttribute).toBeDefined();
+                });
+            });
+
+            test.describe('when the component `status` is set to anything but `error`', () => {
+                test('should not render the `aria-invalid` attribute', async ({ mount }) => {
```

Comment:
I think this might be a slightly more accurate test name, what do you think?
```suggestion
                test('should render `aria-invalid=false`', async ({ mount }) => {
```